tooling/templatize/pkg/pipeline/arm.go

```diff
@@ -285,19 +304,19 @@ func doWaitForDeployment(ctx context.Context, client *armresources.DeploymentsCl
 	if step.DeploymentLevel == "Subscription" {
 		// Hardcode until schema is adapted
```

Comment:
@geoberle what needs to be fixed here? I can do it if you let me know the intent